pkg/inventory/policy_test.go

```diff
@@ -157,7 +157,7 @@ func TestCanApply(t *testing.T) {
 		},
 	}
 	for _, tc := range testcases {
-		actual := CanApply(tc.inv, tc.obj, tc.policy)
+		actual, _ := CanApply(tc.inv, tc.obj, tc.policy)
```

Comment:
Shouldn't we verify that the error is nil here?

Reply:
Not necessary. We can to verify the `actual` is the same as expected. The returned error is not always nil.